### PR TITLE
Fix issue in exporter-powerdns

### PR DESCRIPTION
**Pull request by ivancorrales**
There were a couple of problems that are fixed with this PR:

- The exporter-powerdns binary wasn't created.
- The definitions were failing when checking out the established commit.

### CODE REVIEW

File: scripts/fetch_synthesis_definition.sh [hunk 1]
```diff
@@ -16,9 +17,7 @@ for definition in ${DEFINITION_NAMES}; do
       repo_url=${definition_version%%#*}
       repo_commit=${definition#*#}
       cd ${tmp_dir}; \
-        git remote add forked ${repo_url}; \
-        git fetch forked;  \
-        git switch ${repo_commit} -d;
+        git -c advice.detachedHead=false checkout ${repo_commit};
```
Comment (paologallinaharbur) [inline scripts/fetch_synthesis_definition.sh:20]: why on the same line?

Comment (ivancorrales) [reply scripts/fetch_synthesis_definition.sh:20]: Just to remove the detached head advice only for this statement

Comment (gsanchezgavier) [reply scripts/fetch_synthesis_definition.sh:20]: I guess the question of @paologallinaharbur  is related to the `; \` and the one liner scrip, I don't think is necessary here if i'm not mistaken .

Comment (paologallinaharbur) [reply scripts/fetch_synthesis_definition.sh:20]: yes exactly!

Comment (ivancorrales) [reply scripts/fetch_synthesis_definition.sh]: Done!

**Review by paologallinaharbur — APPROVED** [review]

**Pull request merged**